aip/general/0162.md

```diff
@@ -359,50 +316,22 @@ message DeleteBookRevisionRequest {
   // The name of the book revision to be deleted, with a revision ID explicitly
   // included.
   //
-  // Example: publishers/123/books/les-miserables@c7cfa2a8
+  // Example: publishers/123/books/les-miserables/revisions/c7cfa2a8
   string name = 1 [
     (google.api.field_behavior) = REQUIRED,
     (google.api.resource_reference) = {
-      type: "library.googleapis.com/Book"
+      type: "library.googleapis.com/BookRevision"
     }];
 }
 ```
 
-- The request message **must** have a `name` field to identify the resource
-  revision being deleted.
-  - The explicit revision ID **must** be required (the method **must** fail
-    with `INVALID_ARGUMENT` if it is not provided, and **must not** default to
-    the latest revision).
-  - The field **should** be [annotated as required][aip-203].
-  - The field **should** identify the [resource type][aip-123] that it
-    references.
-- The API **must not** overload the `DeleteBook` method to serve both purposes
-  (this could lead to dangerous or confusing mistakes).
-- If the resource supports soft delete, then revisions of that resource
-  **should** also support soft delete.
-- The method **must not** cause the _resource_ to be deleted.
-  - Because a resource **should not** exist with zero revisions, the method
-    **must** fail with `FAILED_PRECONDITION` if the user attempts to delete the
-    only revision.
-- The response message **must** be the resource itself. There is no
-  `DeleteBookRevisionResponse`.
-  - The response **should** include the fully-populated resource unless it is
-    infeasible to do so.
-
-### Character Collision
-
-Most resource names have a restrictive set of characters, but some are very
-open. For example, Google Cloud Storage allows the `@` character in filenames,
-which are part of the resource name, and therefore uses the `#` character to
-indicate a revision.
-
-APIs **should** avoid permitting the `@` character in resource names, and if
-APIs that do permit it need to support resources with revisions, they
-**should** pick an appropriate separator depending on how and where the API is
-used, and **must** clearly document it.
+- Resource revisions **should** be automatically deleted on parent resource deletion.
+- If resource revisions meant to have longer lifespan than the parent resource, resource and resource revision
+should be side by side resource collections.
 
 ## Changelog
```

Comment:
Please include a rationale and history section to explain the change. Example: https://google.aip.dev/203#rationale.

I think I'd primarily talk about how the goal is to make revisions more of a sub-resource than an extension of an existing resource.